setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name="pyotp",
-    version="2.2.6",
+    version="2.2.7",
```

Comment:
Though a better nit here is bumping a version number in a PR (Since you don't know what the version will be in the master branch when it's finally merged in)

Reply:
It's better 👍   But still [nit](https://en.wiktionary.org/wiki/nit#English) 👎
_[nitpicker](https://en.wiktionary.org/wiki/nitpicker) - One who [nitpicks](https://en.wiktionary.org/wiki/nitpick) (finds fault in **unimportant** details)._

Reply:
Version numbers are very important.